playwright/tests/page-objects/form-editor.ts

```diff
@@ -2,7 +2,7 @@ import type { Locator, Page } from '@playwright/test';
 import { Toolbar } from './toolbar';
 import { Canvas } from './canvas';
 import { Inscription } from './inscription';
-import { v4 as uuid } from 'uuid';
+import { randomUUID } from 'crypto';
```

Comment:
just for my understanding: you are switching a third-party UUID generator to built-in that provides the same.

:heavy_check_mark:  less deps.

Reply:
yes crypto is a package from node. So only works on the server (for playwright in this case)

Reply:
thanks; good to know.